product-details/scss/style.scss

```diff
@@ -0,0 +1,19 @@
+/*------------------------------------*\
+  #ABSTRACTS
+\*------------------------------------*/
+
+@import "../../shared/scss/shared.scss";
+// @import "variables";
```

Comment:
If variables module is not needed for now I think removing the code is cleaner instead of commenting it out :)

Reply:
You can commit it, If you don't need it